userspace/libsinsp/events/sinsp_events_ppm_sc.cpp

```diff
@@ -310,6 +328,24 @@ std::unordered_set<std::string> libsinsp::events::sc_set_to_names(const libsinsp
 	return ppm_sc_names_set;
 }
 
+std::unordered_set<std::string> libsinsp::events::sc_set_to_event_names(const libsinsp::events::set<ppm_sc_code>& ppm_sc_set)
```

Comment:
Not so sure we need this :thinking: maybe we could add it only when we will need it, WDYT?

Reply:
Falco will use this for printing out the selected events. It makes sense, because those will be the same values that Falco will use in both the rulesets and the custom base syscall set in the config file.